sklearn/ensemble/tests/test_forest.py

```diff
@@ -597,7 +597,7 @@ def test_forest_oob_warning(ForestEstimator):
         estimator.fit(iris.data, iris.target)
 
 
-@pytest.mark.parametrize("ForestEstimator", FOREST_CLASSIFIERS_REGRESSORS.values())
+@pytest.mark.parametrize("ForestEstimator", FOREST_CLASSIFIERS.values())
```

Comment:
We also need to add a new test to check that we can compute the OOB score with the regressor since it was the regression in the PR.